registry/userGroups.xml

```diff
@@ -45,4 +45,7 @@
 
 	<!-- Subscription Manager Role -->
 	<group roleId="0x00200000" stages="" name="default.groups.name.subscriptionManager" plural="default.groups.plural.subscriptionManager" abbrev="default.groups.abbrev.subscriptionManager" />
+
+	<!-- Editorial Board Member Role -->
+	<group roleId="0x00001001" stages="" name="default.groups.name.editorialBoardMember" plural="default.groups.plural.editorialBoardMember" abbrev="default.groups.abbrev.editorialBoardMember" masthead="true" />
```

Comment:
(I mentioned that we're going to decimal numbers elsewhere -- that should apply here too, though we haven't done that yet. Leave this to use hex for now, as decimal numbers aren't supported here yet, and we'll clean it up eventually.)